client/src/pages/Login/useStyles.ts

```diff
@@ -8,14 +8,20 @@ const useStyles = makeStyles(() => ({
     },
   },
   authWrapper: {
+    display: 'flex',
```

Comment:
Since these styles have been directly applied to the `<Box>` component, I think they can be removed here?